src/Spago/PackageSet.hs

```diff
@@ -226,10 +238,9 @@ freeze = do
 ensureFrozen :: Spago m => m ()
 ensureFrozen = do
   echoDebug "Ensuring that the package set is frozen"
-  rawPackageSet <- liftIO $ Dhall.readRawExpr path
-  case rawPackageSet of
-    Nothing -> echo "WARNING: wasn't able to check if your package set file is frozen"
-    Just (_header, expr) -> do
-      let areRemotesFrozen = foldMap isRemoteFrozen expr
-      unless (and areRemotesFrozen) $ do
-        freeze
+  imports <- liftIO $ Dhall.readImports "spago.dhall"
```

Comment:
CI is failing because in `Spago.PscPackage` we also call `ensureFrozen`, and in there we only have a `packages.dhall` and not a `spago.dhall`

I suggest we add a `FilePath` parameter to `ensureFrozen`, so that we can pass `configPath` or `packagesPath` where needed